packages/core/src/types.ts

```diff
@@ -1,5 +1,8 @@
 import type { DescribeQuery, ExprNode, Query } from '@uwdata/mosaic-sql';
 
+export * from './util/selection-types.js';
```

Comment:
We should move these exports to index-types.ts instead. I don't think we need them here.

We could also move the Activatable interface definition into this file.